### PR TITLE
feat: Add hotkey default backend implementation

### PR DESCRIPTION
**Pull request by ReaperMaga**

### CODE REVIEW

File: .github/workflows/windows_build.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        uses: actions/upload-artifact@v3
+        with:
+          name: reskey
+          path: build/bin/reskey.exe
+
+      - name: upload amd64 installer
+        uses: actions/upload-artifact@v3
+        with:
+          name: reskey-amd64-installer
+          path: build/bin/reskey-amd64-installer.exe
```
Comment (zoey-kaiser) [inline .github/workflows/windows_build.yml]: Please add a newline at the end of the file.

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,5 @@ build/bin
 node_modules
 frontend/dist
 .idea/
+test.json
+build/windows/installer/tmp
```
Comment (zoey-kaiser) [inline .gitignore]: Please add a newline at the end of the file.

File: build/windows/installer/wails_tools.nsh [hunk 3]
```diff
@@ -5,19 +5,19 @@
...
-    !define INFO_COMPANYNAME "{{.Info.CompanyName}}"
+    !define INFO_COMPANYNAME "reskey"
 !endif
 !ifndef INFO_PRODUCTNAME
-    !define INFO_PRODUCTNAME "{{.Info.ProductName}}"
+    !define INFO_PRODUCTNAME "reskey"
 !endif
 !ifndef INFO_PRODUCTVERSION
-    !define INFO_PRODUCTVERSION "{{.Info.ProductVersion}}"
+    !define INFO_PRODUCTVERSION "1.0.0"
```
Comment (zoey-kaiser) [inline build/windows/installer/wails_tools.nsh:17]: I don't think we are in production yet. Lets change it to:

```suggestion
    !define INFO_PRODUCTVERSION "0.1.0-alpha.1"
```

File: build/windows/installer/wails_tools.nsh [hunk 4]
```diff
@@ -5,19 +5,19 @@
...
 !endif
 !ifndef INFO_COPYRIGHT
-    !define INFO_COPYRIGHT "{{.Info.Copyright}}"
+    !define INFO_COPYRIGHT "Copyright........."
```
Comment (zoey-kaiser) [inline build/windows/installer/wails_tools.nsh:20]:
```suggestion
    !define INFO_COPYRIGHT "Copyright Averix 2023"
```

**Review by zoey-kaiser — CHANGES_REQUESTED** [review]

File: build/windows/installer/wails_tools.nsh (hunk 3 above)
Comment (ReaperMaga) [reply build/windows/installer/wails_tools.nsh:17]: Those are auto generated values, this type of suggestion needs to be applied to wails.json
I will do that.

**Review by zoey-kaiser — DISMISSED** [review]

File: .github/workflows/windows_build.yml [hunk 5]
```diff
@@ -0,0 +1,68 @@
+name: Windows Build
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```
Comment (zoey-kaiser) [inline .github/workflows/windows_build.yml:7]: We will need to see if we want to keep this for every PR, or only on merge. But out of scope, lets look into this in a new PR.

**Review by zoey-kaiser — APPROVED** [review]

**Pull request merged**